### PR TITLE
feat: add page view traces for / and /about

### PR DESCRIPTION
**Pull request by vm**

### CODE REVIEW

**Review by yyip-dev — COMMENTED** [review]
What do you mean by constants from analytics-events? Also, after adding tracing, how do we validate it is working?

**Comment by vm** [comment]
I actually updated it, and I just tested it locally

**Comment by yyip-dev** [comment]
@vm Nice! And out of curiosity how do we test analytics changes?

File: src/pages/Landing/index.tsx [hunk 1]
```diff
@@ -152,7 +153,7 @@ export default function Landing() {
   if (landingPageFlag === LandingPageVariant.Control || !isOpen) return null
 
   return (
-    <>
+    <Trace page="landing-page" shouldLogImpression>
```
Comment (yyip-dev) [inline src/pages/Landing/index.tsx:156]: Should this be referred to by a constant too, i.e. `PageName.LANDING_PAGE`?

**Review by yyip-dev — APPROVED** [review]

File: src/pages/Landing/index.tsx (hunk 1 above)
Comment (vm) [reply src/pages/Landing/index.tsx:156]: yes

**Comment by vm** [comment]
I just console log it manually. AFAIK we don't have a test environment atm...

**Pull request merged**